### PR TITLE
Mock Functions

### LINKED ISSUE #7: Add a VueJS layer & mock some functions
This is going to be a fairly simple first part of the issue: 

- [ ] Add a VueJS layer

What I mean here is that I'll add a reference to the public VueJS library ( as seen in the tutorials here: https://laracasts.com/series/learn-vue-2-step-by-step )

For the second part of this issue I want to mock some custom functions in a JavaScript file. By this I mean to create the function names themselves, without making them work. Knowing which functions we will need ahead of time is part of good application planning and also lends itself to Test Driven Development (we'll want to have some tests in place as the complexity of the app grows - more on that in the future).

Here are some of the functions we'll need:
- [ ] validate spaces (should probably decide on a naming convention, `spaces` or `squares`?)
- [ ] move chip
- [ ] capture chip
- [ ] double-capture
- [ ] reset board

### PR DESCRIPTION
**Pull request by ghost**
fixes #7

### CODE REVIEW

**Comment by ghost** [comment]
@sxkosone I've made some adjustments in this branch to streamline our naming conventions.

I've updated anything that was named "piece" to "chip" and anything named "space" or "box" to "square". 

I've done this in the HTML, CSS and JavaScript.

File: js/chips.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+	// things to check to find out what the available moves are for the selected chip:
+	// - get the array of nextValidSquares from the square's markup (or encapsulate that data into another object somewhere?)
+	// - check each of the nextValidQuares for empty squares or enemy chips/friendly chips using the squareContains function
+	// - create an array of available empty valid squares that the chip can move to
+	// - create an array of valid squares that contain enemy chips
+	// - when a valid square contains an enemy chip, check for available capture
+	// - if a capture is available, create an array of valid squares to *jump-to* that would *capture* an enemy chip
+}
+
+function squareContains(selectedChip) {
```
Comment (sxkosone) [inline js/chips.js:41]: Not sure if I understand this part. Shouldn't the function squareContains receive a parameter of square(s)? Such as squareContains(validateMoves) _parameter is an array, check for each in array_ or squareContains(validSquare) _checks for contents of a valid square_.

The chip doesn't contain information about the next valid moves yet, if I remember correctly. Or will the selectedChip contain that info in the future, like stored in an temporary object?

**Review by sxkosone — APPROVED** [review]
Mock functions looking good! I left one question about squareContains().

File: js/chips.js (hunk 1 above)
Comment (ghost) [reply js/chips.js:41]: In this case my thought process was that there would be an array of `nextValidSquares` and that at some point we'd loop through the array and run a `squareContains()` function for each of those squares in the array. You're right that the function argument would take the ID of a square and not the `selectedChip`, I probably was just on a copy/paste run when I put that argument here 😬

**Pull request merged**